app/analysis/index/controller.js

```diff
@@ -120,10 +120,10 @@ export const AnalysisQueryParams = new QueryParams({
     replace: true,
     refresh: true,
     serialize(val) {
-      return val === "-date" ? null : `${val},id`;
+      return val?.endsWith(",id") ? val : `${val},id`;
```

Comment:
Wasn't the ` ? null` workaround to prevent the qp from resetting when editing reports?

Reply:
yeah, that's at least what the comment states. But could not confirm this while testing with this new code. :shrug:

Reply:
Huh, we spent so many hours trying to fix this and now it just works :sob:

Reply:
But wont `val?.endsWith(",id") ` always be false since the `,id` part is only serialized?

Reply:
> Huh, we spent so many hours trying to fix this and now it just **doens't** work sob

Sad! Indeed.